PiWriteBack.Setup/ProductInstallFiles-all.wxs

```diff
@@ -0,0 +1,565 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
The `ProductInstallFiles*` files can (and probably should) be gitignore'd. They are generated by the build.